### PR TITLE
[SIL-85] Error screen UI

### PR DESCRIPTION
**Pull request by nowakweronika**
### Description
Add error screen UI and extract outlined button to reuse in the application.

### Why?
We want to show user info if something goes wrong :)

### Demo
Screenshot 2023-03-07 at 15 29 20

### Documentation
- Link to task on jira: **[SIL-85](https://silentus-au.atlassian.net/browse/SIL-85)**
- Link to figma: **[Figma](https://www.figma.com/file/yNbcX8VTwufnf7HacYz5Zg/Material-3-Design-Kit-(Community)?node-id=53295%3A27471&t=qooL1FbuUvTroMyU-0)**

### CODE REVIEW

File: app/src/main/java/com/appunite/loudius/ui/components/LoudiusOutlinedButton.kt [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun LoudiusOutlinedButton(
+    onClick: () -> Unit,
```
Comment (kezc) [inline app/src/main/java/com/appunite/loudius/ui/components/LoudiusOutlinedButton.kt]: nitpick: maybe we could place this parameter on the end so we place use lambda outside parentheses

File: app/src/main/java/com/appunite/loudius/ui/components/LoudiusErrorScreen.kt [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+@Composable
+fun LoudiusErrorScreen(
+    errorText: String,
+    buttonText: String,
+    onButtonClick: () -> Unit,
+) {
+    Column(
+        modifier = Modifier.padding(top = 142.dp).fillMaxSize(),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.spacedBy(57.dp),
```
Comment (kezc) [inline app/src/main/java/com/appunite/loudius/ui/components/LoudiusErrorScreen.kt]: nitpick: I think we should even value for margins https://github.com/appunite/Loudius/pull/20#discussion_r1122888216

**Review by kezc — APPROVED** [review]

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -11,4 +11,6 @@
     <string name="error_dialog_title">Error</string>
     <string name="error_dialog_text">Something went wrong…</string>
     <string name="ok">OK</string>
+    <string name="error_image_desc">error image</string>
+    <string name="try_again_text">Try again</string>
```
Comment (Krzysiudan) [inline app/src/main/res/values/strings.xml:15]: **Question:** What is the purpose of adding suffix `text` at the end? 🤔

**Review by Krzysiudan — COMMENTED** [review]
Lgtm!

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (nowakweronika) [reply app/src/main/res/values/strings.xml:15]: It made sense to me yesterday :) But I can change it to just `try_again` in next PR.

**Pull request merged**